client/src/components/HealthCard/HealthCard.jsx

```diff
@@ -0,0 +1,219 @@
+import React, { useState, useEffect } from "react";
+import useStyles from "./styles";
+import {
+  Box,
+  Card,
+  CardContent,
+  Grid,
+  Typography,
+  Divider,
+  IconButton,
+  List,
+  ListItem,
+} from "@material-ui/core";
+import VisibilityIcon from "@material-ui/icons/Visibility";
+import logo from "../../../images/mitre-logo.svg";
+
+var QRCode = require("qrcode");
```

Comment:
Can we change this to use import to remain consistent?
```suggestion
import QRCode from "qrcode";
```

Reply:
updated